communities/aston-css/community-impact-proposal.md

```diff
@@ -0,0 +1,43 @@
+
+I plan to create aston-css/code-of-conduct repo and either a repo for the manifesto (like aston-css/manifesto) or just use our [new website](https://astoncss.com/) and tweak the words to be more specific, like a manifesto. I hope to get both of these done pretty soon, maybe within the next month or so.
+
+### Programme of Events
+
+Continuing with the plan of running regular events, I'd like to come up with a well thought-out plan of talks and workshops. The aim of these will be to draw in interest from more people, and to accomplish this the events will be marketed as helping members achieve something (eg, building a personal site on GitHub pages). I think events are more likely to appeal to people if they know what they're going to get out of them - an actual _thing_ on top of new knowledge.
+
+If we plan out the entire term of events (at least a rough plan) in advance, we can do more ambitious things, and build on top of previous sessions to cover more advanced topics. So we'd lay out the foundations at the start of the year - things like Git, using the terminal, etc.
+
+As mentioned before, we have many members to take advantage of in terms of running talks and workshops. I said that we could have a CFP within our community, and I think this would work really well along with a bit of a nag to specific members!
+
+The summer will act as the planning phase for this, and the entirety of term 1 will be the execution phase.
+
+## Planning for the Future
+
+Certainly with the documentation, this is going to have a lasting effect. The manifesto acts as the definition of what the society is all about, and should help drive decisions that this committee and future committees make. It's also important that this is an evolving document which can be edited and commented on.
+
+I don't think that the docs necessarily have a measurable impact, so it's difficult to work out how successful they are. Perhaps a good idea would be for the committee to review the docs twice a year - once with the new incoming committee, and then once again around mid-term. Members should also be actively encouraged to review them.
+
+With the programme of events, the immediate impact will be seen by the number of members attending the events. In the longer-term, the impact will be the skills and knowledge which members pick up from these sessions and go on to develop further. This is again hard to measure, but if at least one person is encouraged to go on and use what they learn in something, then I'll count it as a success! If the format and planning of the events goes well for the first term, then we'll iterate on the concept and adapt it for people's needs in second term.
```

Comment:
We spoke a little bit about one way you can measure this: proportion of students participating in events that go on to become mentors/speakers at future events. You could also do something nifty with the GitHub API and see which of them goes on to make an active repo from the event ;)